src/main/java/uk/gov/laa/ccms/caab/controller/submission/ClientCreateSubmissionInProgressController.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.laa.ccms.caab.controller.submission;
+
+import static uk.gov.laa.ccms.caab.constants.SessionConstants.CLIENT_DETAILS;
+import static uk.gov.laa.ccms.caab.constants.SessionConstants.CLIENT_INFORMATION;
+import static uk.gov.laa.ccms.caab.constants.SessionConstants.CLIENT_SEARCH_CRITERIA;
+import static uk.gov.laa.ccms.caab.constants.SessionConstants.SUBMISSION_TRANSACTION_ID;
+import static uk.gov.laa.ccms.caab.constants.SessionConstants.USER_DETAILS;
+
+import jakarta.servlet.http.HttpSession;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Controller;
+import org.springframework.util.StringUtils;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.SessionAttribute;
+import uk.gov.laa.ccms.caab.service.ClientService;
+import uk.gov.laa.ccms.data.model.UserDetail;
+import uk.gov.laa.ccms.soa.gateway.model.ClientDetail;
+import uk.gov.laa.ccms.soa.gateway.model.ClientStatus;
+
+/**
+ * Controller for client creation submissions in progress.
+ */
+@Controller
+@RequiredArgsConstructor
+@Slf4j
+public class ClientCreateSubmissionInProgressController {
+
+  private final ClientService clientService;
+
+  /**
+   * Handles the GET request for the client creation submission in progress screen.
+   *
+   * @return The view name for the submission in progress.
+   */
+  @GetMapping("/submissions/client-create")
+  public String submissionsInProgress(
```

Comment:
Should this be non-plural? submissionInProgress.
Based on the name I'd think it was going to display a screen listing all submissions